python/ray/dag/compiled_dag_node.py

```diff
@@ -2056,6 +2056,21 @@ def _get_execution_results(
                 del self._result_buffer[execution_index]
         return result
 
+    def _release_output_channel_buffers(self, execution_index: int):
```

Comment:
Typically, if a class method starts with `_` in the Ray codebase, it is expected to be called only by other methods within the same class. How about removing `_` if it is called in `compiled_dag_ref.py`?